### PR TITLE
Update scale-tidb-using-tiup.md

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16743
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -127,7 +127,29 @@ This section exemplifies how to add a TiDB node to the `10.0.1.5` host.
 
     If you see `Scaled cluster <cluster-name> out successfully`, the scale-out operation succeeds.
 
-3. Check the cluster status:
+3. Refresh the cluster configuration:
```
Comment (qiancai) [inline scale-tidb-using-tiup.md:130]:
```suggestion
3. Refresh the cluster configuration.
```

File: scale-tidb-using-tiup.md [hunk 2]
```diff
@@ -307,7 +329,29 @@ This section exemplifies how to remove a TiKV node from the `10.0.1.5` host.
 
     If you see `Scaled cluster <cluster-name> in successfully`, the scale-in operation succeeds.
 
-3. Check the cluster status:
+3. Refresh the cluster configuration:
```
Comment (qiancai) [inline scale-tidb-using-tiup.md:332]:
```suggestion
3. Refresh the cluster configuration.
```

File: scale-tidb-using-tiup.md [hunk 3]
```diff
@@ -127,7 +127,29 @@ This section exemplifies how to add a TiDB node to the `10.0.1.5` host.
...
+
+        ```shell
+        tiup cluster reload <cluster-name> --skip-restart
+        ```
+
+    2. Refresh the Prometheus configuration and restart Prometheus:
+
+        > **Note:**
+        >
+        > If you are using TiUP v1.15.0 or a later version, you can skip this step. If you are using a TiUP version earlier than v1.15.0, you need to execute the following command to update the Prometheus configuration and restart Prometheus.
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:146]: How about shorter?
```suggestion
        > If you are using TiUP v1.15.0 or a later version, skip this step. If you are using a TiUP version earlier than v1.15.0, execute the following command to update the Prometheus configuration and restart Prometheus.
```

File: scale-tidb-using-tiup.md [hunk 4]
```diff
@@ -307,7 +329,29 @@ This section exemplifies how to remove a TiKV node from the `10.0.1.5` host.
...
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:348]:
```suggestion
        > If you are using TiUP v1.15.0 or a later version, skip this step. If you are using a TiUP version earlier than v1.15.0, execute the following command to update the Prometheus configuration and restart Prometheus.
```

**Review by lilin90 — APPROVED** [review]
Rest LGTM

**Pull request merged**